midealocal/devices/ed/__init__.py

```diff
@@ -77,9 +83,12 @@ def _use_new_set(self) -> bool:
         # if (self.sub_type > 342 or self.sub_type == 340) else False
         return True
 
-    def build_query(self) -> list[MessageQuery]:
+    def build_query(self) -> list[MessageQuery | MessageQuery01]:
```

Comment:
```suggestion
    def build_query(self) -> list[MessageQuery]:
```

Reply:
if device NOT support `MessageQuery` and ONLY support `MessageQuery01`,  we only use `MessageQuery` with the same name, when device send `MessageQuery` with body 00, it will add `MessageQuery` to `self._unsupported_protocol`, so `MessageQuery` with body 01 will be ignored and can't work anymore.
this is the reason why we need to add it with a different class name